src/SortBy/SortBy.jsx

```diff
@@ -30,10 +36,11 @@ SortBy.propTypes = {
     summary: PropTypes.string,
     unit: PropTypes.number,
   }),
+  dispatch: PropTypes.func.isRequired,
 };
 
 SortBy.defaultProps = {
   movieDetails: null,
 };
 
-export default SortBy;
+export default connect()(SortBy);
```

Comment:
This component has to take value of current sort type and to set active button relying on it